data/002/0102/en_US.json

```diff
@@ -2,12 +2,12 @@
 	"002": {
 		"0102": {
 			"name": "Unknown",
-			"message": "This device's access to online services\nhas been restricted by Nintendo.\n\nFor help, visit support.nintendo.com",
+			"message": "This device's access to online services\nhas been restricted by Nintendo.\n\nFor help, visit support.nintendo.com.",
 			"short_description": "Device is banned",
-			"long_description": "Unknown cause.",
-			"short_solution": "Unknown solution.",
-			"long_solution": "Unknown solution.",
+			"long_description": "Your account or Nintendo 3DS console has been banned from online play.",
+			"short_solution": "Don't do whatever you did to get banned",
```

Comment:
```suggestion
			"short_solution": "Don't do whatever you did to get banned.",
```

Reply:
I know there was a discussion on the server, but I think this should match the long solution.